addOns/llm/src/main/java/org/zaproxy/addon/llm/ExtensionLlm.java

```diff
@@ -53,31 +46,18 @@
  */
 public class ExtensionLlm extends ExtensionAdaptor {
 
-    // The name is public so that other extensions can access it
-    public static final String NAME = "ExtensionLlm";
```

Comment:
Sorry if I was unclear here, I just meant the comment should be removed not the constant

Reply:
no worries, I've removed all comments.